static/app/views/issueDetails/groupEventDetails/groupEventDetailsContent.tsx

```diff
@@ -19,6 +19,7 @@ import EventSpanOpBreakdown from 'sentry/components/events/eventStatisticalDetec
 import EventBreakpointChart from 'sentry/components/events/eventStatisticalDetector/breakpointChart';
 import EventComparison from 'sentry/components/events/eventStatisticalDetector/eventComparison';
 import RegressionMessage from 'sentry/components/events/eventStatisticalDetector/regressionMessage';
+import SpanDiff from 'sentry/components/events/eventStatisticalDetector/spanDiff';
```

Comment:
nit: Could this component be named `AggregateSpanDiff` the reason being is that we seem to have two types of diffing components: single-event and aggregate